docs/source/tensors.rst

```diff
@@ -219,6 +219,7 @@ view of a storage and defines numeric operations on it.
    .. automethod:: cpu
    .. automethod:: cross
    .. automethod:: cuda
+   .. automethod:: cummax
```

Comment:
I guess the reason for not having an inplace version is just that we don't have one for `cumsum`?

Reply:
yeah but do you think we should have an inplace version?

Reply:
Maybe not if it's not needed.
It should be easy enough to add it later.